### ISSUE TITLE
Copy & Paste don't work on Windows when editing a note in 0.18a

### ISSUE THREAD

**Issue by erik-brangs** [tomboy-notes/tomboy-ng#75]
I'm using `0.18a` with build date `2018/08/18` on Windows 64. When editing a note, neither `Ctrl+C` nor `Ctrl+V` work. When not editing a note (e.g. in the search bar), the hotkeys seem to work fine. Using the copy & paste functions via the right-click menu while editing a note also works.

**Comment by erik-brangs** [comment]
`Crtl+X` is also affected.

**Comment by davidbannon** [comment]
Right, that been fixed in the 0.18b binaries now in the 0.18 release.
Just a bit of carelessness I'm afraid.

**Comment by erik-brangs** [comment]
Is there a list of combinations that are handled correctly by KMemo? `Ctrl+End` and `Ctrl+Home` are also affected. When they worked, they jumped to the end to the beginning of the note, respectively.

**Comment by davidbannon** [comment]
Oh dear ! it seemed like such an easy fix. 
No, I don't think the keys KMemo does handled is documented anywhere. I'll have to search through the code. Honestly, where that is happening is the correct place to handle this issue, not up in my code.
Trouble is I use the command line or the mouse, I don't generally use ctrl keys.

**Comment by davidbannon** [comment]
OK, here are the control keys I have been able to identify that do work, won't leave ctrl characters in code, are now re-enabled and.
```
Ctrl - PRIOR     Move to top page
Ctrl - NEXT      Move to bottom page
Ctrl - HOME      Move to top of document
Ctrl - END       Move to end of document
Ctrl - UP        Scroll Up
Ctrl - DOWN      Scroll Down
Ctrl - LEFT      Move a word left
Ctrl - RIGHT     Move a word right
Ctrl - RETURN    Centre screen on cursor
Ctrl - INSERT    Paste
Ctrl - A         Select all of document
Ctrl - Y         Delete Line
Ctrl - X, C, V   Cut, Copy and Paste      
```
The editor also supports a range of combinations of Shift, Alt and Control too. I'll have to go over the source and see just what is there, works and what it actually does.

**Comment by davidbannon** [comment]
Erik, I have not pushed out new binaries with these key strokes enabled. Just checked in the code. If you need that functionality now, please let me know.

**Comment by erik-brangs** [comment]
I'm fine with waiting. Getting copy & paste working again was the most important part of this issue for me.
